Dockerfile

```diff
@@ -1,18 +1,157 @@
-FROM ubuntu:bionic
-MAINTAINER Fmstrat <fmstrat@NOSPAM.NO>
+#
+# To build the RPMs
+#
+FROM rockylinux:8.5 as src
 
-ENV DEBIAN_FRONTEND noninteractive
+#
+# Basic Parameters
+#
+ARG ARCH="x86_64"
+ARG OS="linux"
+ARG VER="4.14.5-9"
+ARG DIST="el8_5"
+ARG LDB_VER="2.3.0-2"
+ARG LDB_DIST="el8"
+ARG PKG="samba"
+ARG SRC="https://dl.rockylinux.org/pub/rocky/8/BaseOS/source/tree/Packages/s/samba-${VER}.${DIST}.src.rpm"
 
-RUN apt-get update && apt-get upgrade -y
+#
+# Some important labels
+#
+LABEL ORG="Armedia LLC"
```

Comment:
`ORG` and `MAINTAINER` should refer to ArkCase LLC versus Armedia.

Reply:
Done!